documentation/asciidoc/computers/camera/rpicam_apps_building.adoc

```diff
@@ -67,7 +67,7 @@ First, install the following `libcamera` dependencies:
 $ sudo apt install -y libboost-dev
 $ sudo apt install -y libgnutls28-dev openssl libtiff5-dev pybind11-dev
 $ sudo apt install -y qtbase5-dev libqt5core5a libqt5gui5 libqt5widgets5
-$ sudo apt install -y meson cmake
+$ sudo apt install -y meson
```

Comment:
We should still include cmake in this list as meson makes use of cmake to find system libraries.